src/sidekick/fr.css

```
@@ -158,6 +158,10 @@
   content: "La génération de l'aperçu a échoué. Veuillez essayer plus tard.";
 }
 
+.hlx-sk-overlay .modal.modal-preview-not-gdoc::before {
+  content: "La prévisualisation a échoué. Doit être un document Google ou une feuille de calcul.";
```

I would keep this consistent with the translation on L158 above (or change both):
```suggestion
  content: "La génération de l'aperçu a échoué. Doit être un document Google ou une feuille de calcul.";
```